Misc/NEWS.d/next/Library/2022-03-18-17-25-57.bpo-46382.zQUJ66.rst

```diff
@@ -0,0 +1,2 @@
+:func:`~dataclasses.dataclass` ``slots=True`` now accounts for slots in base classes.
```

Comment:
One final thing: could this message be improved? I think if someone hasn't been following this issue, then they won't understand what "accounts for" means. Maybe something like "now won't include slots defined in base classes", or something like that.

Reply:
no problem. done.